lib/elixir/lib/file.ex

```diff
@@ -74,6 +74,27 @@ defmodule File do
 
   Check `:file.open/2` for more information about such options and
   other performance considerations.
+
+  ## Seeking within a file
+
+  To read from a specific position in a file, use `:file.pread/3`:
+
+      iex> temp_filename = Path.join(System.tmp_dir!(), "example")
+      iex> File.write(temp_filename, "Eats, Shoots & Leaves")
+      iex> file = File.open!(temp_filename)
+      iex> :file.pread(file, 14, 7)
+      {:ok, " Leaves"}
```

Comment:
To simplify this example, we don't need the doctest style and temporary file. Instead, you can do something like:

```suggestion
      File.write!("/tmp/example.txt", "Eats, Shoots & Leaves")
      file = File.open!("/tmp/example.txt")
      :file.pread(file, 14, 7)
      #=> {:ok, " Leaves"}
```

Thoughts?

Reply:
I was trying to avoid `/tmp` because it's platform-specific. I thought having a copy-pastable example was a better tradeoff even if it's a bit more verbose. I made the doctest change, feel free to put `/tmp/example.txt` back if you think it's better!

Reply:
How about just `example.txt?`